examples/specs/brush_table.vl.json

```diff
@@ -6,7 +6,7 @@
     "window": [{"op": "row_number", "as": "row_number"}]
   }],
   "hconcat": [{
-    "selection": [{"name": "brush", "select": "interval"}],
+    "params": [{"name": "brush", "select": "interval"}],
```

Comment:
I feel like `"select": "interval"` reads kinda weird.  Wouldn't it be clearer if this is `selection`?

I could see that it doesn't make sense as the first commit in https://github.com/vega/vega-lite/pull/6926 because having selection inside selection is kinda weird.

Reply:
I prefer `select` both because it's the verb form (and so reads more naturally, i.e., "select an interval") and because it's consistent with `bind` (rather than `binding`) which occurs at the same level.

Reply:
I agree with @arvind.